src/handler/patch.ts

```diff
@@ -0,0 +1,61 @@
+import 'dotenv/config';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import logger from '../util/logger';
+import { addHttpHeaders } from '../util/httpHeaders';
+import {
+  archiveOldCreateCurrentRecord,
+  getBySystemNumberAndCreatedTimestamp,
+} from '../services/database';
+import { validateUpdateVinRequest, validateVins } from '../validators/patch';
+import { processPatchVinRequest } from '../processors/processPatchVinRequest';
+import { TechrecordGet } from '../models/post';
+import { formatTechRecord } from '../util/formatTechRecord';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  logger.info('Amend VIN Called');
+
+  const isRequestInvalid: APIGatewayProxyResult | undefined = validateUpdateVinRequest(event);
+
+  if (isRequestInvalid) {
+    return isRequestInvalid;
+  }
+
+  logger.debug('Request is Valid');
+
+  const systemNumber: string = decodeURIComponent(event.pathParameters?.systemNumber as string);
+  const createdTimestamp: string = decodeURIComponent(event.pathParameters?.createdTimestamp as string);
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const newVin: string = JSON.parse(event.body!).newVin! as string;
```

Comment:
Can we try some syntax round here to get rid of these non null assertions?

```suggestion
  const newVin: string = JSON.parse(event.body ?? '').newVin ?? '' as string;
```

Or something like that would be even better, or a ternary something like that